src/request.js

```diff
@@ -1,28 +1,28 @@
-const createGame = () => {
+async function createGame() {
   const games = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games';
-  return fetch(games, {
+  const response = await fetch(games, {
     method: 'POST',
     body: JSON.stringify({
       name: 'Montanholi\'s Game',
     }),
     headers: {
       'Content-type': 'application/json; charset=UTF-8',
     },
-  })
-    .then((response) => response.json())
-    .then((json) => json.result.split(' ')[3]);
-};
+  });
+  const result = await response.json();
+  return result.result.split(' ')[3];
+}
```

Comment:
- [ ] Good job I can see you used `await` and `async` but according to the requirements avoid using `function` keyword but instead use `arrow function`